proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -40,6 +40,10 @@ message Repository {
     string organization_id = 6;
   }
   Visibility visibility = 7;
+  // deprecated means this repository is deprecated.
+  bool deprecated = 8;
+  // deprecation_message is the message shown if the repository is deprecated.
+  string deprecation_message = 9;
```

Comment:
I know you mentioned that you wanted to split these out so that users could deprecate something without including a message, but we might actually want to require a message in all cases (similar to what we do for [API tokens](https://buf.build/settings/user)).

Plus, that gives us the benefit that if `deprecation_message` is set, then we know the entity is deprecated. And it's impossible for `deprecated_message` to be set, but have `deprecated` equal to `false.

Reply:
Idk, I might keep them separate anyways, and do backend validation that false + non-empty is never done - I can see it both ways